common/src/java/org/apache/hadoop/hive/conf/HiveConf.java

```diff
@@ -2012,6 +2012,9 @@ public static enum ConfVars {
         "Set to 1 to make sure hash aggregation is never turned off."),
     HIVE_MAP_AGGR_HASH_MIN_REDUCTION_LOWER_BOUND("hive.map.aggr.hash.min.reduction.lower.bound", (float) 0.4,
         "Lower bound of Hash aggregate reduction filter. See also: hive.map.aggr.hash.min.reduction"),
+    HIVE_MAP_AGGR_HASH_FLUSH_SIZE_PERCENT("hive.map.aggr.hash.flush.size.percent", (float) 0.1,
+        "Whether percentage of the map side group aggregation hash tables to flush.　\n" +
```

Comment:
@ryukobayashi, could you please update the description before the merge
`Percentage of entries to flush in the map-side group aggregation hash table`

Reply:
Got it. will do.